ci-support.sh

```diff
@@ -258,13 +258,13 @@ pip_install_project()
 
   # Append --editable to PROJECT_INSTALL_FLAGS, if not there already and if not
   # disabled.
-  if ! test -f setup.cfg || ! grep -q disable-editable-pip-install setup.cfg; then
+  if test -f setup.cfg && ! grep -q disable-editable-pip-install setup.cfg; then
     if [[ ! $PROJECT_INSTALL_FLAGS =~ (^|[[:space:]]*)(--editable|-e)[[:space:]]*$ ]]; then
         PROJECT_INSTALL_FLAGS="$PROJECT_INSTALL_FLAGS --editable"
     fi
   fi
 
-  if ! test -f pyproject.toml || ! grep -q disable-editable-pip-install pyproject.toml; then
+  if test -f pyproject.toml && ! grep -q disable-editable-pip-install pyproject.toml; then
```

Comment:
Want to use the toml wrangler from https://github.com/inducer/ci-support/commit/c14e73ddb0deb84d3d99efc81920b0f2c69bc6ec? :grin:

Reply:
Should be done in https://github.com/inducer/ci-support/compare/2206e53e68e01d6d6722d9449fa2b8ae21034ea1..1029d2c589c03b95214316b2da6e6ac42e3a821b